gallery/how_to/work_with_microtvm/micro_aot.py

```diff
@@ -125,7 +125,7 @@
 #
 # Now that we have the compiled model as an IRModule, we need to create a firmware project
 # to use the compiled model with microTVM. To do this, we use Project API. We have defined
-# CRT and Zephyr microTVM template projects which are used for x86 CPU and Zephyr platforms
+# CRT and Zephyr microTVM template projects which are used for x86 CPU and Zephyr boards
 # respectively.
```

Comment:
I'm not a native English speaker, but I think that ideally there must be a comma before "respectively", so it's up to you to add it or not :) I won't block on this nit, so just saying in case you need to re-spin the PR after some other review comment and if you can confirm that is indeed correct ;)

Reply:
there should be, for some reason my eyes where seeing it there but I didn't actually put it there lol

Reply:
haha I missed it previously too! :)